cupyx/scipy/sparse/_csc.py

```diff
@@ -341,7 +341,7 @@ def transpose(self, axes=None, copy=False):
                 Otherwise, it shared data arrays as much as possible.
 
         Returns:
-            cupyx.scipy.sparse.spmatrix: Transpose matrix.
+            cupyx.scipy.sparse.csr_matrix: Transpose matrix in CSR format.
```

Comment:
Would you use the same description as scipy? (The part after colon)

Reply:
copied from https://github.com/scipy/scipy/blob/686422c4f0a71be1b4258309590fd3e9de102e18/scipy/sparse/_base.py#L768